### ISSUE TITLE
t/Plack-App-Data-Printer/03-general.t failures (with older Data::Printer?)

### ISSUE THREAD

**Issue by eserte** [michal-josef-spacek/Plack-App-Data-Printer#1]
On some of my smoker systems the test suite fails:
```
#   Failed test 'Get content (data structure).'
#   at t/Plack-App-Data-Printer/03-general.t line 31.
#          got: '\ {
#     baz   [
#         [0] 1,
#         [1] 2
#     ],
#     foo   "bar"
# }
# '
#     expected: '{
#     baz   [
#         [0] 1,
#         [1] 2
#     ],
#     foo   "bar"
# }
# '
# Looks like you failed 1 test of 3.
t/Plack-App-Data-Printer/03-general.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
```
Statistical analysis suggests that this happens with older Data::Printer (Theta != 1 means "bad"):
```
****************************************************************
Regression 'mod:Data::Printer'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.0000	   0.00
[1='eq_0.38']  	      0.0000	      0.0000	   0.00
[2='eq_0.40']  	      0.0000	      0.0000	   0.00
[3='eq_0.99_020']	      0.0000	      0.0000	   0.00
[4='eq_1.000004']	      1.0000	      0.0000	19067202136181300.00

R^2= 1.000, N= 74, K= 5
****************************************************************
```

**Comment by michal-josef-spacek** [comment]
@eserte Do you know which version of Data::Printer is it? I tested bunch of versions, but without success.

**Comment by michal-josef-spacek** [comment]
Ok, i know.
Data::Printer <= 0.40 has this issue.
Data::Printer > 0.40 has ANSI sequences in output.
Data::Printer > 1 is ok

**Comment by michal-josef-spacek** [comment]
Fixed in 0.03 release (97fe0b909a3bb51f95665f91ea21985931aba423)
